sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultClientTest.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@EnabledIfEnvironmentVariable(named = "AZURE_KEYVAULT_CERTIFICATE_NAME", matches = ".*")
+public class KeyVaultClientTest {
+    private KeyVaultClient keyVaultClient;
+    private String certificateName;
+
+    String getEnvValue(String key) {
+        String value = System.getenv(key);
+        System.out.println("*****************************logStart");
```

Comment:
please remove these system.out.println from code, and if you need to log information during runtime, use Logging with right output level.

having such system.out.println in the code ( test code is also production code ) is very unprofessional